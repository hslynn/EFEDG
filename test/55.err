Running /share/soft/mvapich2-2.0.1_icc/bin/mpiexec.hydra -np 55 -ppn 28 -f /share/home/hslynn/.lsbatch/1546156965.208199.hostfile -env MV2_CPU_BINDING_POLICY scatter ./simplest
