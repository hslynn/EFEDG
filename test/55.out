Sender: LSF System <lsfadmin@c1u26n03>
Subject: Job 208199: <55> in cluster <hpc.lsec.cc.ac.cn> Exited

Job <55> was submitted from host <ln02> by user <hslynn> in cluster <hpc.lsec.cc.ac.cn>.
Job was executed on host(s) <36*c1u26n03>, in queue <batch>, as user <hslynn> in cluster <hpc.lsec.cc.ac.cn>.
                            <19*c1u24n03>
</share/home/hslynn> was used as the home directory.
</share/home/hslynn/Einstein_DG/test> was used as the working directory.
Started at Sun Dec 30 16:06:17 2018
Results reported on Sun Dec 30 16:16:11 2018

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
mpijob -t mvapich2 ./simplest
------------------------------------------------------------

TERM_OWNER: job killed by owner.
Exited with signal termination: Killed.

Resource usage summary:

    CPU time :                                   31.25 sec.
    Max Memory :                                 211 MB
    Average Memory :                             211.00 MB
    Total Requested Memory :                     -
    Delta Memory :                               -
    Max Processes :                              28
    Max Threads :                                28

The output (if any) follows:



PS:

Read file <55.err> for stderr output of this job.

